win32ss/gdi/ntgdi/freetype.c

```diff
@@ -2688,6 +2688,7 @@ GetFontFamilyInfoForList(LPLOGFONTW LogFont,
     FONTFAMILYINFO InfoEntry;
     DWORD Count = *pCount;
 
+    /* for all entries in the list */
```

Comment:
I think that this was obvious from the next line...

Reply:
Removed the comment.